daq/docker_test.py

```diff
@@ -40,11 +40,15 @@ def start(self, port, params, callback):
                     "GATEWAY_IP=" + params['gateway_ip'],
                     "GATEWAY_MAC=" + params['gateway_mac']]
 
+
         if 'local_ip' in params:
             env_vars += ["LOCAL_IP=" + params['local_ip'],
                          "SWITCH_PORT=" + params['switch_port'],
                          "SWITCH_IP=" + params['switch_ip']]
 
+        if 'switch_model' in params:
```

Comment:
is there a situation where switch_model exists without the other parameters?   If not, let's fold this into the previous if clause. Also, I think we might as well make this a required parameter since the results are meaningless without it.

Reply:
This should be resolved now.